OpenRA.Game/Graphics/SpriteLoader.cs

```diff
@@ -21,6 +22,12 @@ public interface ISpriteLoader
 		bool TryParseSprite(Stream s, out ISpriteFrame[] frames);
 	}
 
+	public interface ISpriteLoader2 : ISpriteLoader
```

Comment:
just replace 
```
IcnD2Frame[] ParseFrames(Stream s)
```
 with 
```
IcnD2Tile[] ParseFrames(Stream s)
```
and remove everything in this function after this comment
```
// first loop. calculate final array size
```
at the end just return tiles array. And you now will not see buildings in asset browser. only buildings parts. is this what you want?